src/renderers/webxr/WebXRController.js

```diff
@@ -8,13 +8,47 @@ function WebXRController() {
 
 	this._targetRay = null;
 	this._grip = null;
+	this._hand = null;
 
 }
 
 Object.assign( WebXRController.prototype, {
 
 	constructor: WebXRController,
 
+	getHandSpace: function () {
+
+		if ( this._hand === null ) {
+
+			this._hand = new Group();
+			this._hand.matrixAutoUpdate = false;
+			this._hand.visible = false;
+
+			this._hand.joints = [];
+			this._hand.inputState = { pinching: false };
+
+			if ( window.XRHand ) {
+
+				for ( let i = 0; i <= XRHand.LITTLE_PHALANX_TIP; i ++ ) {
```

Comment:
It's best to use `window.XRHand` here to in order to get rid of the linter warnings.

Reply:
Not sure where the linter warnings come from, but they don't complain about `XRWebGLLayer`, ...

Reply:
I've added `XRHand` to `utils/build/externs.js`.

Reply:
It seems eslint (`npm run test-lint`) still complains.

Reply:
Well, maybe they will go automatically away when eslint is upgraded in the next time 😅 .

Reply:
We still might want to fix these warnings since they now pop up in each new PR.